### ISSUE TITLE
'Meta' object has no attribute 'app_config'

### ISSUE THREAD

**Issue by zruu** [stephenmcd/mezzanine#1628]
I don't know if this is a issue with Mezzanine or Constance. But here we go I got this traceback below when trying to go to **/admin** :

`'Meta' object has no attribute 'app_config'`

It refers to the function admin_app_list() in **/core/templatetags/mezzanine.tags**

**Comment by stephenmcd** [comment]
Constance does some strange things where it registers an admin class with a fake model/meta class, that causes the breakage:

https://github.com/jazzband/django-constance/blob/master/constance/admin.py

I'm going to push a change that works around it, but you should raise an issue with that project.

**Comment by stephenmcd** [comment]
BTW Mezzanine itself already has database-driven settings, so you may also want to re-evaluate what you're doing.

http://mezzanine.jupo.org/docs/configuration.html

**Comment by stephenmcd** [comment]
Oh, I see you opened an issue there already.

https://github.com/jazzband/django-constance/issues/148

**Comment by zruu** [comment]
Yeah, I understand that I can use Mezzanine to register settings. I like that way. But I want to register settings to a whole new settings page for users that do not have permission to change SSL settings and so forth. I don't know how to accomplish that with Mezzanine.
